code_schemes/ws_correct_dataset.json

```diff
@@ -176,6 +188,15 @@
       "NumericValue": -40,
       "StringValue": "NR",
       "VisibleInCoda": false
+    },
+    {
+      "CodeID": "code-CE-016c1e22",
+      "CodeType": "Control",
+      "ControlCode": "CE",
+      "DisplayText": "CE (coding error)",
+      "NumericValue": -60,
```

Comment:
Noise_other_project code has - 60 numeric value. Can we change this?

Reply:
Yes,  I realised this and switch to -110 elsewhere. Is that an ok number?

If so I will correct here

Reply:
-110 is okay. Thanks